content/js1/sprints/2/prep/index.md

```diff
@@ -8,4 +8,406 @@ backlog= 'Module-JS1'
 backlog_filter= 'Week 2'
 +++
 
+## Learning objectives
 
+- Label and describe the syntactic features of a function declaration
+- Identify a call site
+- Differentiate between a function declaration and a function invocation
+- Propose a function definition to satisfy some problem requirements
+- Given a short program, determine which variable names are _bad_ choices
+- Give examples of when the `undefined` data type may occur in a JavaScript program
+- Construct simple expressions to represent a set of operations
+- Define a _parameter_
+- Identify the return value a call site given its function declaration
+- Identify the value assigned to a parameter when a function is invoked
+- Identify the type of a return value given some call site
+- Given a short program, state the number of times a given function is called
+- Differentiate between parameters and arguments
+- Invoke a given function with appropriate arguments to produce some target output
```

Comment:
We've only covered a single parameter/argument so far. I'd either make this explicitly singular, or show an example with multiple parameters.